data/Fedora/devel/bootstrap/aarch64/runtime-binary-packages-full.txt

```diff
@@ -1999,6 +1988,7 @@ lvm2-dbusd-2.02.174-2.fc28.aarch64
 lvm2-devel-2.02.174-2.fc28.aarch64
 lvm2-libs-2.02.174-2.fc28.aarch64
 lvm2-lockd-2.02.174-2.fc28.aarch64
+lxpolkit-0.5.3-2.fc27.3.aarch64
```

Comment:
NAK!  Something else (e.g. gnome-shell or gnome-settings-daemon) should be providing PolicyKit-authentication-agent.

Reply:
I think it is polkit-gnome

Reply:
Nowadays it's provided by gnome-shell:

https://git.gnome.org/browse/gnome-shell/tree/src/shell-polkit-authentication-agent.c